### DISCUSSION TITLE
How do I get embeddings for a particular face on a photo? Is there a way to pass coordinates?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [exadel-inc/CompreFace#825] [Q&A] [answered]
How do I get embeddings for a particular face on a photo? Is there a way to pass coordinates (using python CompreFace)

**Comment by pospielov** [comment, marked as answer]
if you know the coordinates of the face, then you can just crop the face and send it to CompreFace.
By default, CompreFace returns the embeddings of the N biggest faces, the result contains coordinates of bounding boxes.

**Reply by martinenkoEduard** [reply]
N biggest faces..
Is there to change that?

**Reply by pospielov** [reply]
How do you want to change it?

**Reply by martinenkoEduard** [reply]
I mean what is the default N biggest faces value? Is there a 100 faces on the photo, in default settings will it return all of the faces?

**Reply by pospielov** [reply]
the default value is 1.
If you set it to -1, it will return all detected faces. 
The parameter is called `limit`
Please check the documentation: 
https://github.com/exadel-inc/CompreFace/blob/master/docs/Rest-API-description.md#face-detection-service

**Reply by martinenkoEduard** [reply]
Got it 

| optional | maximum number of faces on the image to be recognized. It recognizes the biggest faces first. Value of 0 represents no limit. Default value: 0
